src/main/java/uk/gov/companieshouse/web/accounts/exception/NavigationException.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.companieshouse.web.accounts.exception;
+
+/**
+ * The class {@code NavigationException} is a form of {@link RuntimeException}
+ * that is thrown if errors occur when attempting to determine if
+ * a conditional controller should render or not during navigation.
+ *
+ * @see uk.gov.companieshouse.web.accounts.service.navigation.NavigatorService
+ * @see uk.gov.companieshouse.web.accounts.controller.ConditionalController
+ **/
+public class NavigationException extends RuntimeException {
+
+    /**
+     * Constructs a new {@code NavigationException} with a custom message and the specified
+     * cause.
+     *
+     * @param message a custom message
+     * @param cause the cause
+     */
+    public NavigationException(String message, Throwable cause) {
+        super(cause);
```

Comment:
Should we call `super(message, cause)` so that we don't lose the provided message?